.travis.yml

```diff
@@ -57,7 +57,8 @@ install:
   - git clone https://github.com/openresty/openresty.git ../openresty
   - git clone https://github.com/openresty/openresty-devel-utils.git
   - git clone https://github.com/simpl/ngx_devel_kit.git ../ndk-nginx-module
-  - git clone https://github.com/openresty/lua-nginx-module.git ../lua-nginx-module
+  - git clone https://github.com/hnakamur/lua-nginx-module.git ../lua-nginx-module
```

Comment:
You can use the `-b BRANCH` option o fthe `git clone` command here BTW.

Reply:
Thanks for the tip! I didn't know that.
But it's temporary until the https://github.com/openresty/lua-resty-core/pull/141 is merged, so I'll leave it as is.